__tests__/app.test.js

```diff
@@ -165,4 +162,53 @@ describe("news-api", () => {
             })
         })  
     })
+    describe("POST requests", () => {
+        describe("POST /api/articles/:article_id/comments", () => {
+            test("responds with status code 201 and the posted object in expected format", () => {
+                return request(app)
+                .post("/api/articles/1/comments")
```

Comment:
This is spot-on, but we could do with an additional test to check whether the new post has actually made it's way into the database. Once the new comment has been posted, try utilising your GET comments endpoint and assert that you expect the new comment to be included.